content/2017.03.building-projects-as-a-way-to-learn.rst

```diff
@@ -0,0 +1,222 @@
+
+Note that, it might not be so cheap anymore when you start adding
+documentation and maintenance costs.
+
+The product might be used, and might get some traction for months or
+even years but it will be eventually forgotten in a git repository
+somewhere forever.
+
+Writing software to solve problems is not enough, to keep motivated on
+the long term. You need a greater goal than just writing code.
+
+Knowing that, how do you keep motivated building things that are
+thrown away?
+
+Different people or communities might have different answers.
+
+For us the answer is: As well as solving the problem, what can we
+learn while building this solution that can be useful for us and for
+our community to help us go faster and build more robust software
+later.
+
+For Mozilla the answer to that question is: How do I improve Firefox
+so that people want to use it as their main browser?
+
+Both answer lead to goals that we set to ourselves. We are using
+software development to reach them.
+
+Mozilla is calling that the True North, we are calling it a vision,
+some other people might call it a utopia.
+
+In both cases that vision should set the route of our ambition and give
+people a aim to all look in the same direction.
+
+
+How do you turn this vision into OKRs?
+======================================
+
+OKR means **Objectives and Key results**: it is a way to define what
+you want to achieve in a tangible way and what you will be able to
+mesure
+`to see how effective you were <https://blog.betterworks.com/okrs-stretch-or-not-to-stretch/>`_.
+
+It is also a way to cut your vision in steps and to move your vision
+forward.
+
+
+How do you set course for a vision?
+===================================
+
+To be able to define OKRs from your vision, you need to be able to find
+the fastest way to get closer to your aim, however what we can learn
+from sailing is that the fastest way is rarely the shortest.
+
+At Mozilla, we have a vision that is written down in the
+`Mozilla Manifesto <https://www.mozilla.org/en-US/about/manifesto/>`_.
+
+Mitchell Baker does a great job making sure everyone at Mozilla stays
+aligned with that vision.
+
+However agreeing on the fastest way and turning the Mozilla vision
+into OKRs is really challenging and even more when you have a thousand
+people's opinion about it.
+
+It's the board's job to set course every six months (or even less) on
+what we should be focusing on for the next six months.
+
+When this happens, it often means changing our course and killing
+projects or at least putting them on the side if teams are turning to
+new priorities.
+
+At our level, we are driven by the use case. We have a vision of what
+we want to build and we keep adding ideas in the backlog.
+
+However we are not building anything until we have a proper use case
+for it. As soon as we have a use case it is added to our Roadmap.
+
+You endup not spending time on tasks you don't need and for each use
```

Comment:
`end up`. I also think there's something missing before this paragraph. You say we have a vision of what we want to build, but I still don't really see how you cut that vision into "ideas" that go into the backlog, or how the idea differs from a use case, or how the backlog differs from a roadmap, and how we don't work on anything that's not on the roadmap...